version/version.go

```diff
@@ -12,7 +12,7 @@ var (
 	HyperShiftImage = "registry.ci.openshift.org/hypershift/hypershift:latest"
 )
 
```

Comment:
how did you figured this url? Is there anything we can add in a comment so others have context and can fix it if this happens again in the future?

Reply:
https://docs.ci.openshift.org/docs/getting-started/useful-links/#services

Reply:
Added comment